internal/api/mfa.go

```diff
@@ -272,11 +272,11 @@ func (a *API) VerifyFactor(w http.ResponseWriter, r *http.Request) error {
 				output.Message = hooks.DefaultMFAHookRejectionMessage
 			}
 
-			return forbiddenError(output.Message)
+			return forbiddenError(ErrorCodeMFAVerificationRejected, output.Message)
 		}
 	}
 	if !valid {
-		return badRequestError("Invalid TOTP code entered")
+		return unprocessableEntityError(ErrorCodeMFAVerificationFailed, "Invalid TOTP code entered")
```

Comment:
Potential legitimate HTTP status code change, though unlikely to be handled by developers.

Reply:
Should be OK.